contributing/code/standards.rst

```diff
@@ -160,6 +160,9 @@ Structure
   ``tearDown`` methods of PHPUnit tests, which should always be the first methods
   to increase readability;
 
+* Declare all the arguments in the same line as the method/function name, no
```

Comment:
@javiereguiluz 

Something like this, inspired from @WouterJ link.

Right:

```php
class ClassName extends ParentClass implements \ArrayAccess, \Countable
{
    // constants, properties, methods
}
```

Wrong:

```php
class ClassName extends ParentClass implements
    \ArrayAccess,
    \Countable,
    \Serializable
{
    // constants, properties, methods
}
```